metrics/pipe_tasks.yaml

```diff
@@ -0,0 +1,29 @@
+# Alert Production metric definitions in pipe_tasks
```

Comment:
My intent was that these would evolve into section headers:

    # Alert Production metric definitions in pipe_tasks
    [code]
    # Data Release Production metric definitions for pipe_tasks
    [code]

Is that not a good way of organizing the metrics within each package?